plugins/woocommerce/includes/shortcodes/class-wc-shortcode-checkout.php

```diff
@@ -297,10 +297,19 @@ private static function order_received( $order_id = 0 ) {
 			return;
 		}
 
+		/**
+		 * Indicates if known (non-guest) shoppers need to be logged in before we let
+		 * them access the order received page.
+		 *
+		 * @param bool $verify_known_shoppers If verification is required.
+		 *
+		 * @since 8.4.0
+		 */
+		$verify_known_shoppers = apply_filters( 'woocommerce_order_received_verify_known_shoppers', true );
 		$order_customer_id = $order->get_customer_id();
```

Comment:
```suggestion
		$verify_known_shoppers = apply_filters( 'woocommerce_order_received_verify_known_shoppers', true );
		$order_customer_id     = $order->get_customer_id();
```

(To satisfy the linter's equal-signs-must-be-aligned rule.)